javascript/autoFillSearchParams.js

```diff
@@ -0,0 +1,45 @@
+class AutoFillSearchParams {
+    searchParams = new URLSearchParams(decodeURIComponent(location.search));
+    result = '';
+    paramsKeys = ['prompt', 'negative_prompt', 'size', 'seed', 'steps', 'sampler', 'cfg', 'clip_skip', 'batch_size', 'ensd']
```

Comment:
Based on the requirement, it should be `width` and `height` specifically.

Reply:
I tried width and height, but it don't work correctlly，size does work